### DISCUSSION TITLE
日志频繁显示ndex obj count: XXX

### DISCUSSION THREAD

**Discussion by commandoccz** [AlistGo/alist#4568] [General]
最近想起来看看日志，结果发现日志中频繁显示INFO[2023-06-13 12:15:40] index obj count: 1 
简直就是刷屏，请问有没有大神知道这是神马个情况，简直看不到其他日志信息了，以前都没有这样刷屏的
![image](https://github.com/alist-org/alist/assets/76011848/6d947081-a0ce-4dc0-bc07-669cc55cb968)

**Comment by anwen-anyi** [comment]
这是你开启了自动更新索引吧？应该是

**Reply by commandoccz** [reply]
还真是
![image](https://github.com/alist-org/alist/assets/76011848/d8dc5eef-6313-43d4-a86a-a549dfc295c5)

**Reply by commandoccz** [reply]
但是有个问题，我这文件也不止1517个啊，日志里面index obj count最高都到4万多了

**Reply by xhofe** [reply]
后台中的对象计数仅显示完全重建索引时的文件数量，后续的自动更新索引不会重新计数。

**Reply by commandoccz** [reply]
最后一个问题，我如何屏蔽info呢，我的是docker，貌似alist运行命令是固定在docker里面的
config.json文件可以控制日志等级么？
![image](https://github.com/alist-org/alist/assets/76011848/e026e188-02e8-44e5-8b75-acd9b49aee99)
